test/reply-early-hints.test.js

```diff
@@ -88,7 +84,7 @@ test('sends early hints (http2)', (t) => {
     })
 
     req.on('end', () => {
-      t.equal(data, testResBody)
+      t.assert.strictEqual(data, testResBody)
       client.close()
       fastify.close(t.end)
```

Comment:
`t.end` is a `tap` method we should remove this
```suggestion
      fastify.close()
```